packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache

```diff
@@ -1,8 +1,10 @@
 <div class="card">
-  <header class="hidden" id="fxa-connect-another-device-header"><h1 class="card-header mb-8">{{#t}}Connect another device{{/t}}</h1></header>
+  <header id="fxa-connect-another-device-header" class="{{^isFirefoxDesktop}}hidden{{/isFirefoxDesktop}}"><h1 class="mb-8 text-left text-base">{{#t}}Connect another device{{/t}}</h1></header>
   {{#showSuccessMessage}}
     {{#isSignedIn}}
-      <h2 id="fxa-connected-heading" class="card-header mb-8 before:content-circle-check"><span class="align-top ltr:pl-3 rtl:pr-3">{{#t}}You’re signed into Firefox{{/t}}</span></h2>
+      {{^isFirefoxDesktop}}
+        <h2 id="fxa-connected-heading" class="card-header mb-8 before:content-circle-check"><span class="align-top ltr:pl-3 rtl:pr-3">{{#t}}You’re signed into Firefox{{/t}}</span></h2>
```

Comment:
Can we use [`ps-3`](https://tailwindcss.com/docs/padding) instead of `ltr:pl-3 rtl:pr-3`?